running_an_event.md

```diff
@@ -0,0 +1,60 @@
+
+The first key part is to follow the checklist in the [Event Checklist](#event_checklist) section below. Making sure that everything is in place before the event means a smoother process for organizers, and helping things go smoother on the day of the event itself.
+
+The second key part is to publicize the event well. This means getting on Twitter and advertising the event regularly. It's best to start of slowly when the event is first announced, and ramping up the posting frequency until the day of the event. The week of the event, there should be about one tweets per day about it. Each tweet should be different, but always make sure to mention the day, speaker, topic, and venue. Each topic doesn't have to be mentioned in each tweet though, and indeed mixing up which are mentioned is a great way to diversify the tweets. Each tweet should always include a link to the meetup page for the event though. Here are some examples:
+
+![Example Tweets](https://theoreticalideations.com/static/example_nodebots_tweets.png)
+
+The final key part is to have at least two people helping to run the event the day of. Things can and will go wrong that couldn't be foreseen ahead of time, and having two people there means one can make a last minute run to grab cables, batteries, a WIFI router (we actually had to do that once), etc, while the other person stays at the venue.
+
+## Event Checklist
+
+For each event, copy the following checklist code into the GitHub issue:
+
+```
+Event Checklist:
+
+- [ ] Speaker selected: <Calculate 6 weeks prior>, unassigned
+- [ ] Venue secured: <Calculate 4 weeks prior>, unassigned
+- [ ] Meetup.com event created: <Calculate 3 weeks prior>, unassigned
+- [ ] Hardware ordered (if applicable): <Calculate 2 weeks prior>, unassigned
+- [ ] Sponsor found for hardware: <Calculate 2 weeks prior>, unassigned
+- [ ] Venue and hardware sponsor added to meetup.com sponsors list <Calculate 2 weeks prior>, unassigned
+- [ ] Food and drink: <Calculate 1 week prior>, unassigned
+    - [ ] Will there be a good selection of non-alcoholic drinks provided? A proper selection will include more than just water and a token bottle of soda
+    - [ ] Vegetarian options are required, gluten free and vegan are nice to haves.
+- [ ] Hardware purchases reimbursed: <Calculate 1 week prior>, unassigned
+- [ ] Code of Conduct incident reporter test: <Calculate 2 days prior>, unassigned
+    - [ ] Send test email to incident@nodebotssf.org and make sure all recipients received the test
+    - [ ] Send test text message to (415) 944-4941 and make sure al recipients received the test
+```
+
+Each item in the checklist has a due date associated with it. Make sure to calculate the exact dates and replace them in the checklist. Also make sure that each task gets assigned to someone.
+
+## Meetup.com page template
+
+Note: this is in markdown format, but will need to be converted in the Meetup.com editor.
```

Comment:
Thanks for the tip @brntbeer! I don't think we can use it here, since we can't get the HTML into meetup.com, but I'm definitely bookmarking that package for later use :)